app/templates/components/components/header/template.njk

```diff
@@ -44,61 +44,63 @@
 
 <!-- usa header -->
 <header class="usa-header usa-header--extended">
-  <div class="usa-navbar">
-    <div class="usa-logo display-flex flex-align-center" id="-logo">
-      {# <div class="logo-img display-flex">
-        <span class="usa-sr-only">US Notify Logo</span>
-        <image src="{{ params.assetsPath | default('/static/images') }}/us-notify-color.png" alt="US Notify logo" xlink:href=""
-          class="usa-flag-logo margin-right-1" width="40" height="35"></image>
-      </div> #}
-      <em class="logo-text usa-logo__text">
-        <a href="/" title="notify.gov">notify.gov</a>
-      </em>
+  <div class="usa-nav-container">
+    <div class="usa-navbar">
+      <div class="usa-logo display-flex flex-align-center flex-justify" id="-logo">
+        {# <div class="logo-img display-flex">
+          <span class="usa-sr-only">US Notify Logo</span>
+          <image src="{{ params.assetsPath | default('/static/images') }}/us-notify-color.png" alt="US Notify logo" xlink:href=""
+            class="usa-flag-logo margin-right-1" width="40" height="35"></image>
+        </div> #}
+        <em class="logo-text usa-logo__text">
+          <a href="/" title="notify.gov">notify.gov</a>
```

Comment:
do we need capital N?

Reply:
That's a good catch, @terrazoon!

@maomeara63 or @stvnrlly, we have been using `Notify.gov` across the board, right?

Reply:
got it